src/utils/input_validation/validators.py

```diff
@@ -0,0 +1,177 @@
+import os, sys, uuid 
+
+def validate_room_number(person_id: str) -> bool: 
+    """
+    Check if the user input for room entity is valid
+    """
+    if person_id.isnumeric():
+        return 0 <= int(person_id) <= 9999
+    return False
+
+
+def validate_guest_id(person_id: str) -> bool:
+    """
+    Check if the user input for guest entity is valid.
+    Return proper exit code for special user id's
+    """
+
+    if person_id.isnumeric():
+        return "0" <= person_id <= "9999"
+    return False
+
+
+def validate_credit_card_number(credit_card_number: float) -> bool:
+    """
+    Check if the user input for credit card
+        follows the international card standards.
+    """
+    if credit_card_number.isnumeric():
+        return 8 <= len(credit_card_number) <= 19
+    return False
+
+
+def is_phone_number_valid(phone_number: str):
+    "Check if the user input for phone number is valid"
```

Comment:
In Python, docstrings are usually housed in triple-quote style """your-documentation""", thus, I'd suggest changing it.